nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you also include the `method name` and the `class name` in the error message to help user to know what is exactly the error once it occurs. You can update it to something like this:
```ruby
   def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
```

Reply:
Of course. 👍